plugins/link-show-direction.js

```diff
@@ -1,17 +1,24 @@
 // @author         jonatkins
 // @name           Direction of links on map
 // @category       Tweaks
-// @version        0.2.1
+// @version        0.2.2
 // @description    Show the direction of links on the map by adding short dashes to the line at the origin portal.
 
 
+/* exported setup --eslint */
+/* global L, dialog, addHook, map, links */
```

Comment:
There is no sense to define them in EVERY file.
Either global definition (for some of entities), or prefix with `window`.

Reply:
it was decided NOT to avoid window. wher possible.
as the exclusions/globals may differ from file to file a global definition is inconvenient. Same with splitting these